packages/palette_generator/test/palette_generator_test.dart

```diff
@@ -70,6 +70,7 @@ void main() async {
     // cache for the other tests, but they timeout if they too are testWidgets
     // tests.
     tester.pumpWidget(const Placeholder());
+    return new Future<Null>.value();
```

Comment:
This looks fishy. Rationale? Can we just make the closure async?

Reply:
No, it already is async, but testWidgets requires a function that returns a Future<Null>.  This was to fix an analyzer finding that showed up because the analyzer got more precise.

Reply:
Sorry, I misunderstood. Yes, the closure is now async.